packages/vm/src/modules/file_system_cache.rs

```diff
@@ -149,7 +162,8 @@ mod tests {
         // Check the returned module is functional.
         // This is not really testing the cache API but better safe than sorry.
         {
-            let cached_module = cached.unwrap();
+            let (cached_module, module_size) = cached.unwrap();
+            assert_eq!(module.serialize().unwrap().len(), module_size);
```

Comment:
```suggestion
            assert_eq!(module_size, module.serialize().unwrap().len());
```

Let's try to keep the actual value left and the expected value right for consistency. I know, Rust makes no difference and in Go the expected value is left and the actual is right. But I think consistency helps reading this a lot.